src/Components/article-comments.jsx

```diff
@@ -0,0 +1,62 @@
+import { useContext, useEffect, useState } from "react";
+import UserContext from "../Contexts/user-context";
+import { getCommentsByArticleId, getUsers } from "../../api";
+
+
+function ArticleComments(props){
+
+    const { currentUser } = useContext(UserContext)
+    const [comments, setComments] = useState([])
+    const articleId = props.articleId
+    
+
+    useEffect(()=>{
+        getCommentsByArticleId(articleId)
+        .then((response)=>{
+            setComments(response.comments)
+        })
+    }, [currentUser])
+
+
+
+
+
+
+    return (
+        
+        <ul id="comment-list">
+            {comments.map((comment)=>{
+                return (
+                    <div className='comment' key={comment.comment_id}>
+                    
+                        
+                        <h3 className='comment-header'>{comment.author}</h3>
+                        <p >{comment.body}</p>
+                        {currentUser!==null? 
+                            <section className='comment-buttons'>
+                                {comment.author===currentUser.username?
+                                    <> 
+                                        <p>{comment.votes}</p>
+                                        <button>Delete My Comment</button>
+                                    </>:
+                                    <>
+                                        <p>{comment.votes}</p>
+                                        <button className='comment-button'>Vote Up!</button>
+                                        <button className='comment-button'>Vote Down!</button>
```

Comment:
This task is a bit too early to put the vote buttons on, since you're not yet giving them any functionality yet, it's better to keep them in a separate PR (with full functionality!)